docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc

```diff
@@ -31,10 +31,9 @@ Once configured, we can execute the Helm command to install the Cloudflow enterp
 
 [source,shell script,subs="attributes+"]
 ----
-helm upgrade cloudflow-enterprise-components cloudflow-helm-charts/cloudflow-enterprise-components \
-  --install \
+helm install cloudflow-enterprise-components cloudflow-helm-charts/cloudflow-enterprise-components \
   --namespace cloudflow \
-  --set enterpriseOperatorVersion={cloudflow-version} \
```

Comment:
Didn't this work for you? It would make sure to always show the current Cloudflow version.

Reply:
Agreed.
I just copied, what was there.

Reply:
Fixed it, it was unintended